client/a8c-for-agencies/sections/signup/agency-details-form/hooks/use-signup-form-validation.ts

```diff
@@ -0,0 +1,71 @@
+import { useTranslate } from 'i18n-calypso';
+import { useCallback, useState } from 'react';
+import { AgencyDetailsPayload } from '../types';
+
+export const CAPTURE_URL_RGX =
+	/^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([-.][a-z0-9]+)*\.[a-z]{2,63}(:[0-9]{1,5})?(\/.*)?$/i;
```

Comment:
Even though I'm not particularly fond of regular expressions because they can be hard to understand, and URLs can be formatted in many obscure ways, I think it is fine to use one in this case because I don't believe agencies will use obscure URL formats for websites.

Reply:
Yeah, I haven't found anything better I could re-use within calypso.